templates/app_page.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+
+{% block content%}
+    <div class="content">
+        <!-- TODO: change href -->
+        <div class="kbcb-back-link"><a href="{{url_for('get_apps')}}"><i class="fa fa-chevron-left"></i> back to the Catalog</a></div>
+        <!-- {{ app }} -->
+        <div class="app-header">
+            <div class="app-header-icon">
+                {% if app.icon %}
+                <img class="icon-img-l" src="https://kbase.us/services/narrative_method_store/{{ app.icon.url }}" >
```

Comment:
This url should be configured somewhere. What happens if this isn't running on production?